source/slang/slang-compiler.h

```diff
@@ -2835,6 +2835,8 @@ namespace Slang
         };
         Dictionary<TargetRequest*, RefPtr<TargetInfo>> m_targetInfos;
 
+        CompilerOptionSet m_optionSetForDefaultTarget;
```

Comment:
Why can't we use m_defaultTarget.optionSet?

Reply:
We can use it, but still we can only copy it. It's life cycle will be end after parseOptions() call.

Reply:
Update the change by using copy the optionSet instead of call inheritFrom().

Reply:
I see.